test/commands/generate_test.rb

```diff
@@ -59,6 +59,7 @@ def default_options
           content.must_match %(  class Index)
           content.must_match %(    include Web::Action)
           content.must_match %(    def call(params))
+          content.wont_match %(      self.body = 'OK')
```

Comment:
@AlfonsoUceda nope, that's why I test that self.body is not present in the resultant action. Note the ```wont_match```